locale/zh/messages.json

```diff
@@ -12618,6 +12640,15 @@
       ]
     ]
   },
+  "{runningInstances, plural, one {# instance running out of {instancesTotal}} other {# instances running out of {instancesTotal}}}": {
```

Comment:
Does this belong? Was it another forgotten extraction?

Reply:
Unless I'm missing something I think this goes with https://github.com/dcos/dcos-ui/blob/90853ef9285377a235d11eb428888a47d79b44de/plugins/services/src/js/components/ServiceStatusProgressBar.js#L22-L27